packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -101,6 +102,10 @@ class BaseAwsSdkPlugin extends ClientPlugin {
     // implemented by subclasses, or not
   }
 
+  addSpanPointers (span, response) {
```

Comment:
Is this empty method needed on the parent class?

Reply:
Yes, since we call `this.addSpanPointers(span, response)` in `apm:aws:request:complete`. Seems silly right now since S3 is the only service using it, but we will be expanding span pointers to Dynamo very soon, and eventually other services.